Source/Core/Common/FileUtil.cpp

```diff
@@ -755,8 +772,10 @@ std::string GetSysDirectory()
 	sysDir = GetBundleDirectory() + DIR_SEP + SYSDATA_DIR;
 #elif defined (_WIN32)
 	sysDir = GetExeDirectory() + DIR_SEP + SYSDATA_DIR;
+#elif defined (LINUX_LOCAL_DEV)
+	sysDir = GetExeDirectory() + DIR_SEP + "Sys";
```

Comment:
Its a relative directory path on Windows, so it might have been redefined based on the platform defines. Maybe check there?